apps/mobile/src/components/Feed/Posts/PostCreatorAndCollectionSection.tsx

```diff
@@ -113,35 +119,36 @@ export function PostCreatorAndCollectionSection({ tokenRef }: Props) {
             handlePress={handleUsernamePress}
             pfpDisabled
           />
-        </View>
+        </GalleryTouchableOpacity>
       ) : null}
 
       {token.community && (
-        <View className="flex" style={contractNameContainerStyle}>
+        <GalleryTouchableOpacity
+          className="flex"
+          style={contractNameContainerStyle}
+          onPress={handleCommunityPress}
+          eventElementId="Post Community Link"
```

Comment:
```suggestion
          eventElementId="Post Collection Link"
```

Reply:
small detail, lets change Community -> Collection to match the label. I know the internal code is all called community but gonna try to migrate the terminology where we can!